runconfig/hostconfig.go

```diff
@@ -96,38 +109,46 @@ func (n PidMode) Valid() bool {
 	return true
 }
 
+// DeviceMapping represents the device mapping between the host and the container
 type DeviceMapping struct {
 	PathOnHost        string
 	PathInContainer   string
 	CgroupPermissions string
 }
 
+// RestartPolicy represents the restart policies of the container
 type RestartPolicy struct {
 	Name              string
 	MaximumRetryCount int
 }
 
+// IsNone indicates whether the container has no restart policy
```

Comment:
I wonder if this would be better to say:
```IsNone indicates whether the container's restart policy is set to "no".```
The current wording can be read to mean "there is no policy defined at all", and I don't think that's correct.  Probably a poorly named func name.